tests/baselines/reference/user/npmlog.log

```diff
@@ -0,0 +1,106 @@
+Exit Code: 1
+Standard output:
+../../../../node_modules/@types/gulp-help/index.d.ts(70,15): error TS2430: Interface 'GulpHelp' incorrectly extends interface 'Orchestrator'.
```

Comment:
This is in _every_ test, but isn't actually in the tests' respective `node_modules` folder. can we fix this?

Reply:
Yeah, I think these tests are missing the `typeRoots` override our other tests have.

Reply:
Ah, that's right. I'll add that.